libs/designsystem/src/lib/components/list/list.component.integration.spec.ts

```diff
@@ -174,4 +174,107 @@ describe('ListComponent', () => {
       expect(lastItem).toHaveComputedStyle({ 'padding-bottom': '0px' });
     });
   });
+
+  fdescribe('when a list have 1 element', () => {
```

Comment:
Beware of all the fdescribes! Needs to be removed before merge.

Reply:
removed